config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:post_id", to: "posts#show"
```

Comment:
- [ ] Great job on your routes; however, they (posts routes) don't align with the URLs outlined in the project description. Take a look at the comparison below 👇 and please make the necessary adjustments.

**_Your routes_**
![image](https://github.com/forLoop94/blogApp/assets/106119105/b44962b7-87d6-4ec2-be65-d791c4df3940)

_**Given routes**_
![image](https://github.com/forLoop94/blogApp/assets/106119105/d9cdba9e-8cd9-4611-84a8-78074c282283)


- [OPTIONAL] I think it would be better to use [resource routing](https://guides.rubyonrails.org/v3.2.2/routing.html#resource-routing-the-rails-default) for defining routes. It not only simplifies your code but also aligns with Rails conventions, making your application more maintainable and developer-friendly. 

```ruby
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```